doc/content/sqa/theory_manual.md

```diff
@@ -0,0 +1,3 @@
+# TMAP8 Theory Manual
+
+Documentation for all the theory supporting TMAP8 is contained in the [syntax/index.md] page.
```

Comment:
BlackBear did it one way, Malamute had a different way. I just picked approach mostly because I don't want folks to think it's the wrong link when they are directly redirected to the same `syntax/index.md`. Having a separate user and theory page that then point to the `syntax/index.md` achieved that goal.